modules/op-running-unpriveleged-builds-in-a-container.adoc

```diff
@@ -0,0 +1,62 @@
+:_content-type: PROCEDURE
+
+[id='op-running-unpriveleged-builds-in-a-container._{context}']
+= Running unprivileged builds in a container
+
+You can run builds as a non-root user inside a container to restrict specific container activities and enhance security.
+
+[INFO]
```

Comment:
```suggestion
[IMPORTANT]
```
Please use one of [the four supported admonition types](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions).